app/models/order.rb

```diff
@@ -8,6 +8,7 @@ class Order < ActiveRecord::Base
   belongs_to :stockit_contact
   belongs_to :stockit_organisation
   belongs_to :organisation
+  belongs_to :beneficiary, class_name: 'Beneficiary'
```

Comment:
No need for class name here unless it is different from the attribute name. I.e. Rails infers the `Beneficiary` class from the `beneficiary` arg to belongs_to. For the `created_by` below, class_name is set to `User` as there is no `created_by` model.

Reply:
Agree with @steveyken here. We can remove `class_name` here.